src/data/player.py

```diff
@@ -176,7 +340,7 @@ def __eq__(self, other):
         )
 
     def __repr__(self):
-        if self.id == 1:
-            return f'{self.__class__.__name__}(EXEMPT)'
+        if self.ref_id == 1:
+            return f'{self.__class__.__name__}(#{self.id} EXEMPT)'
```

Comment:
I don't think we should still use `EXEMPT`in the representation, we should probably use `PAB`instead:
```suggestion
            return f'{self.__class__.__name__}(#{self.id} PAB)'
```